DXRBalance/DeusEx/Classes/Cat.uc

```diff
@@ -0,0 +1,6 @@
+class DXRCat injects #var(prefix)Cat;
+
+defaultproperties
+{
+     bLittle=True
```

Comment:
Just use their `Mass` instead, shouldn't need most of these files

Reply:
I considered that, but I wanted to make it so there's no way anything hostile can be unable to be knocked out. It wouldn't cause any problems with the normal code, but I felt it was safer to make it a property of the class rather than its mass.